libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+                TestVc.builder().evidence(SUCCESSFUL_EVIDENCE).build(),
+                Instant.ofEpochSecond(1705986521));
+    }
+
+    static String vcPassportM1aFailed() throws Exception {
+        return generateVerifiableCredential(
+                TestVc.builder().evidence(UNSUCCESSFUL_EVIDENCE).build(),
+                Instant.ofEpochSecond(1705986521));
+    }
+
+    static String vcPassportM1aWithCI() throws Exception {
+        return generateVerifiableCredential(
+                TestVc.builder().evidence(D02_CI_EVIDENCE).build(),
+                Instant.ofEpochSecond(1705986521));
+    }
+
+    static String vcPassportM1aMissingEvidence() throws Exception {
+        return generateVerifiableCredential(
+                TestVc.builder().evidence(Collections.emptyList()).build(),
+                Instant.ofEpochSecond(1705986521));
+    }
+
+    static String vcPassportMissingName() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder().name(Collections.emptyList()).build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .credentialSubject(credentialSubject)
+                        .evidence(SUCCESSFUL_EVIDENCE)
+                        .build());
+    }
+
+    static String vcPassportMissingBirthDate() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder().birthDate(Collections.emptyList()).build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .credentialSubject(credentialSubject)
+                        .evidence(SUCCESSFUL_EVIDENCE)
+                        .build());
+    }
+
+    static String vcPassportInvalidBirthDate() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .birthDate(List.of(new BirthDate("invalid")))
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .credentialSubject(credentialSubject)
+                        .evidence(SUCCESSFUL_EVIDENCE)
+                        .build());
+    }
+
+    static String vcPassportMissingPassport() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder().passport(Collections.emptyList()).build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .credentialSubject(credentialSubject)
+                        .evidence(SUCCESSFUL_EVIDENCE)
+                        .build());
+    }
+
+    static String vcAddressOne() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder().address(List.of(ADDRESS_1)).build();
```

Comment:
I initially thought this was going to be a VC from the address CRI, until I realised that it will default to being a passport VC. Wonder if a different method name might make it clearer?